test/test-utils/phase-interceptor.ts

```diff
@@ -379,7 +356,7 @@ export class PhaseInterceptor {
    * @param args - Additional arguments to pass to the original method.
    */
   setMode(mode: UiMode, ...args: unknown[]): Promise<void> {
-    const currentPhase = this.scene.phaseManager.getCurrentPhase();
+    const currentPhase = this.scene.phaseManager.getCurrentPhase()!;
```

Comment:
```suggestion
    // @ts-expect-error: `getCurrentPhase()` won't return `null` here
    const currentPhase = this.scene.phaseManager.getCurrentPhase();
```
This way it'll be flagged for removal when `getCurrentPhase()` is no longer able to return `null`.

Reply:
Oh right, that's not correct lol. One sec.